book.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-undef */
```

Comment:
- [ ] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
I don't understat why the project is not validated.
[Autorisation to desable some rules](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md)

If you read the requirement there is a TIP that allow us to disable some rule And it's just the max class in the same page